cea/demand/metamodel/nn_generator/input_matrix.py

```diff
@@ -359,15 +360,14 @@ def get_array_HVAC_variables(building):
     return array_hvac
 
 
-def run_as_script():
+def main(config):
     gv = cea.globalvar.GlobalVariables()
-    scenario_path = gv.scenario_reference
-    locator = cea.inputlocator.InputLocator(scenario=scenario_path)
+    locator = cea.inputlocator.InputLocator(scenario_path=config.scenario)
     building_name = 'B155066'
```

Comment:
ideally, this would be moved to the configuration file, as the `main` function get's called for the script.

Reply:
This is to be seen after we refactor the neural network in a new issue